now.json

```diff
@@ -1,5 +1,15 @@
 {
-  "version": 1,
+  "version": 2,
```

Comment:
Now version 2 doesn't support docker deployments, that's broken at the moment.

Reply:
Reverted to version 1 in https://github.com/opencollective/opencollective-frontend/pull/1980